### DISCUSSION TITLE
Failed to fetch in Grafana

### DISCUSSION THREAD

**Discussion by niro93** [teslamate-org/teslamate#4035] [General]
Hello together,

i am getting "Failed to fetch" in the upper right corner in grafana all the time.
![image](https://github.com/teslamate-org/teslamate/assets/47181261/5bbc46a1-6cc0-4965-b101-572d15b32047)

Where could I find out why? There are no errors in the Grafana log.

Greetings Nico

**Comment by cwanja** [comment]
What do the logs say?

Do you have an ad blocker or uBlock?

**Comment by niro93** [comment]
Thanks, i disabled ublock for my teslamate and now the error is gone.
Must be something new in ublock.
